lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -279,9 +279,15 @@ function _dosomething_campaign_closed_page_access($node) {
  * Determines whether a user has access to the active page callback.
  */
 function _dosomething_campaign_active_page_access($node) {
-  // Staff only.
-  if (!dosomething_user_is_staff()) { return FALSE; }
+  global $user;
+  // Grant access if you're  staff OR you're an org friend:
+  $has_access = dosomething_user_is_staff() || in_array('org friend', $user->roles);
```

Comment:
In addition, do you think it worth create dedicated `dosomething_user_is_staff_and_org_friend` function or extend `dosomething_user_is_staff` with `$include_org_friends` argument?